src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -514,6 +516,24 @@ HasUnsupportedJoinWalker(Node *node, void *context)
 }
 
 
+/* QueryReferencesView returns true if the query references a view RTE */
+static bool
+QueryReferencesView(Query *subquery)
```

Comment:
This isn't really an accurate name - there very well might not be a single reference to that view anymore.